sdk/include/psdk/ntddndis.h

```diff
@@ -443,3 +486,14 @@ typedef struct _NDIS_CO_LINK_SPEED {
 #endif
 
 #endif /* _NTDDNDIS_ */
```

Comment:
This is the header guard. You generally want everything to be above it.

Reply:
Moved it above the `#endif /* _NTDDNDIS_ */`, I believe that's what you wanted?